### DISCUSSION TITLE
Show last known activity instead of network error in IOS widget

### DISCUSSION THREAD

**Discussion by mjarkk** [community/community#4867] [Mobile] [answered]
So I'm a cheap boi and have to relay on wifi to get new information.
But now whenever I'm outside of my house I see this error and it makes the widget useless and look ugly on my homescreen. 
![image](https://user-images.githubusercontent.com/15320763/127775728-25a6b041-88f5-4848-abd1-b6eab917cbab.png)

I would prefer to rather see the last known activity rather than this as some information is always better than non. Maybe add a small icon to indicate that the data is outdated but some info is always better than non.

**Comment by brianlovin** [comment, marked as answer]
Great feedback, I agree @mjarkk — I've passed this along to the team, and we've opened a tracking issue for this work 😊
